src/distributions/weighted.rs

```diff
@@ -130,6 +132,342 @@ impl<X> Distribution<usize> for WeightedIndex<X> where
     }
 }
 
+/// A distribution using weighted sampling to pick a discretely selected item.
+///
+/// Sampling an [`AliasMethodWeightedIndex<W>`] distribution returns the index
+/// of a randomly selected element from the vector used to create the
+/// [`AliasMethodWeightedIndex<W>`]. The chance of a given element being picked
+/// is proportional to the value of the element. The weights can have any  type
+/// `W` for which an implementation of [`AliasMethodWeight`] exists.
+///
+/// # Performance
+///
+/// Given that `n` is the number of items in the vector used to create an
+/// [`AliasMethodWeightedIndex<W>`], [`AliasMethodWeightedIndex<W>`] will take
+/// up `O(n)` amount of memory. More specifically it takes up some constant
```

Comment:
Should probably be "up to" or "will require".

Reply:
Fixed in 9c44b6aa510b25408ce050d2785b5a9fcb4e98de.